presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -141,6 +153,20 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
         }
     }
 
+    public static DistributedQueryRunner createMaterializingQueryRunner(Iterable<TpchTable<?>> tables)
+            throws Exception
+    {
+        return createQueryRunner(
+                tables,
+                ImmutableMap.of(
+                        "query.partitioning-provider-catalog", "hive",
+                        "query.materialize-exchanges-strategy", "ALL",
+                        "query.hash-partition-count", "5",
+                        "colocated-joins-enabled", "true",
+                        "grouped-execution-for-aggregation-enabled", "true"),
```

Comment:
Also set concurrent lifespan per task to be 1 :)

Reply:
It is `1` by default. Do we want to have it set so explicitly?